docker-compose-releng.yml

```diff
@@ -0,0 +1,84 @@
+version: "2.1"
+
+# TODO: what links do we need?
+services:
+  releng_db:
+    image: postgres:9.6
```

Comment:
I really don't think this is going to be a good way to go. The Makefile is focused around running one app at a time, and assumes that everything is running on the same host. None of this is the case in the Docker setup. We're going to have to do a bunch of extra work to have the Makefile's postgres targets support the Docker environment, including:
- Make it learn how to create users
- Make it possible to create multiple databases (not needed yet, but will be as soon as we have more than one Ship It app with a database)
- Make it possible to conditionally bind to 0.0.0.0.

Basically, we have to more or less fork the postgres targets and use different ones in Docker. I do not think it's worth our time to write or maintain this code when the upstream postgres image will do it for us.

If the goal is reproducability and consistency across all machines, we can even pin to a specific digest of the postgres image, which should alleviate any concerns about things working on one machine and not another:
```
    expose:
      - 5432
    environment:
      - POSTGRES_PASSWORD=shipit
      - POSTGRES_USER=shipit
➜  services git:(docker) ✗ docker-compose -f docker-compose-shipit.yml up
Pulling shipit_db (postgres@sha256:6ac0fbeddde3bb7b0003a2ace8c126f742f8bdd90695801337d3edaaf1fcc478)...
sha256:6ac0fbeddde3bb7b0003a2ace8c126f742f8bdd90695801337d3edaaf1fcc478: Pulling from library/postgres
```